manifests/profile/apt.pp

```diff
@@ -15,6 +15,17 @@
   Optional[Hash] $local_repo = undef,
 ) {
 
+  if($facts['os']['family'] == 'Debian') {
+    if $local_repo {
+      apt::source { 'local':
+        *            => $local_repo,
```

Comment:
This change is so that our local repo is loaded by Ubuntu nodes as well as Debian nodes. Everything else in this file remains Debian-only.